portability-transfer/src/main/java/org/datatransferproject/transfer/logging/EncryptingLayout.java

```diff
@@ -0,0 +1,32 @@
+package org.datatransferproject.transfer.logging;
+
+import java.util.UUID;
+import org.apache.log4j.Layout;
+import org.apache.log4j.helpers.ISO8601DateFormat;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class EncryptingLayout extends Layout {
+
+  static UUID jobId;
+
+  public static void setJobId(UUID inputJobId) {
```

Comment:
Let's check that it is not already set and generate an error if it is attempted to be set twice

Reply:
Done.